### ISSUE TITLE
.blend files import issue in Android editor

### ISSUE THREAD

**Issue by surendra019** [godotengine/godot#92825]
### Tested versions

Using Godot v 4.0 android version

### System information

Android 14

### Issue description

I am not able to import .blend files in Godot android editor because of the blender 3 path. What path should I give there to make it work properly. I have also tried to get the complete blender installation folder into the android device and then given its path, but it also don't work. Please help

### Steps to reproduce

Try to give blender 3 path in Godot android editor

### Minimal reproduction project (MRP)

You can create a new project in android editor

**Comment by bruvzg** [comment]
You can't. There's no Blender for Android, and if there was, Godot won't have permissions to run an executable that is not part of the same app.

**Comment by Zireael07** [comment]
This should be clearly documented and the blender path option made uninteractable on Android

**Comment by bruvzg** [comment]
> blender path option made uninteractable on Android

As well as `fbx2gltf`, `oidn_denoise` and all export tools options that are not usable as well.

**Comment by AThousandShips** [comment]
Do we have any system in place to control editor settings based on platform? Other than completely hiding them?

**Comment by bruvzg** [comment]
> Do we have any system in place to control editor settings based on platform? Other than completely hiding them?

I do not think so, hiding is easy, but not disabling it will likely require adding new `PROPERTY_HINT_`.

**Comment by AThousandShips** [comment]
I think just documenting it more clearly and adding some specific warnings would be sufficient, and just hiding the setting itself

**Comment by bruvzg** [comment]
> I do not think so, hiding is easy, but not disabling it will likely require adding new `PROPERTY_HINT_`.

Actually, I'm wrong, there's `PROPERTY_USAGE_READ_ONLY` that seems to do it (but UI is not super clear, it's only minimal difference in color). And it can have a list of supported platforms in the tooltip line, other platform specific APIs do.

<img width="797" alt="Screenshot 2024-06-06 at 14 29 51" src="https://github.com/godotengine/godot/assets/7645683/e5eda08e-caf5-41dd-b758-a8a2db9dc283">

**Comment by AThousandShips** [comment]
Then we can just have it be marked as read only on unsupported builds to expose the tooltip, to avoid confusion, along with adding a note to [this page](https://docs.godotengine.org/en/latest/tutorials/editor/using_the_android_editor.html)
